### DISCUSSION TITLE
Installation on Oracle Cloud

### DISCUSSION THREAD

**Discussion by g-vijayvergia** [Sunbird-Ed/Community#581] [Documentation] [answered]
Dear All,

Request to provide pointers or  any documentation link available for setting up Sunbird-ED in **_Oracle Cloud (OCI)_** .  I have gone through 
`sunbird-devops -- > private repo  --> ansible --> Inventory --> dev --> core --> common.yml`
there are various properties which are specific to 3 cloud providers (Supported values are aws, azure, gcloud). Not sure what to define for Oracle cloud and will it work on Oracle cloud.

I'm new to Sunbird platform. Any help or guidance is appreciated. 

Thanks & Regards
Gaurav

**Comment by amitpriyadarshi** [comment]
@santhosh-tg @tsprasath can you provide any link for asked details.
cc: @gandham-santhosh @Krishnaj20 @vinukumar-vs

**Reply by gandham-santhosh** [reply]
Hi @g-vijayvergia ,
Thanks for writing to us. OCI compatible scripts are not yet available in our repositories. We are in the process of having these changes merged. 
Meanwhile, you can reach out to @ddevadat @heungheung who were able to setup Sunbird Ed on OCI

**Reply by Krishnaj20** [reply]
@ddevadat @heungheung have successfully deployed ED on OCI. Can you help answer this query?

**Reply by gandham-santhosh** [reply, marked as answer]
@g-vijayvergia OCI changes are merged to 5.1.0 hotfix version of Ed. Please find the release notes information for the same at https://ed.sunbird.org/use/releases/release-notes/5.1.0-hotfix
@vinukumar-vs @ddevadat @heungheung

**Comment by gaurav-vijayvergia** [comment]
Thank you @gandham-santhosh . Will try it out.
